CHANGELOG.md

```diff
@@ -1,3 +1,132 @@
+# Kubermatic 2.17
+
+## [v2.17.0](https://github.com/kubermatic/kubermatic)
+
+### Misc
+
+- New etcd backup and restore controllers [#5668](https://github.com/kubermatic/kubermatic/issues/5668) 
+- Add `kubermatic-seed` stack target to the Installer [#6435](https://github.com/kubermatic/kubermatic/issues/6435) 
+- Add an endpoint to list Vsphere datastores: GET /api/v2/providers/vsphere/datastores [#6442](https://github.com/kubermatic/kubermatic/issues/6442) 
+- KKP Installer will update Helm releases automatically if the values have changed (no need for `--force` in most cases). [#6449](https://github.com/kubermatic/kubermatic/issues/6449) 
+- Introduce resource quota for Alibaba provider [#6458](https://github.com/kubermatic/kubermatic/issues/6458) 
+- Add Gatekeeper health to cluster health status [#6461](https://github.com/kubermatic/kubermatic/issues/6461) 
+- Remove CoreOS Support [#6465](https://github.com/kubermatic/kubermatic/issues/6465) 
+- Relax memory limit for openvpn container [#6467](https://github.com/kubermatic/kubermatic/issues/6467) 
+- Enable VM resource quota for AWS provider [#6468](https://github.com/kubermatic/kubermatic/issues/6468) 
+- Multus has been added as an addon.  [#6477](https://github.com/kubermatic/kubermatic/issues/6477) 
+- CoreDNS version now based on Kubernetes version [#6501](https://github.com/kubermatic/kubermatic/issues/6501) 
+- Add support for &#34;use-octavia&#34; setting in Openstack provider specs. It defaults to &#34;true&#34; but leaves the possibility to set it to &#34;false&#34; if your provider doesn&#39;t support Octavia yet but Neutron LBaaSv2 [#6529](https://github.com/kubermatic/kubermatic/issues/6529) 
+- Add components override field to set nodeportrange for apiserver [#6533](https://github.com/kubermatic/kubermatic/issues/6533) 
+- Extend Kubermatic Constraint Template spec with a selector which enables cluster filtering for EE Kubermatic.  [#6534](https://github.com/kubermatic/kubermatic/issues/6534) 
+- CCM/CSI migration [#6535](https://github.com/kubermatic/kubermatic/issues/6535) 
```

Comment:
This was done for Openstack only, AFAIK maybe being explicit about that in the changelog so that we don't leave misleading comments

Reply:
Removed